doc/classes/AudioStream.xml

```diff
@@ -39,6 +39,12 @@
 				Returns the length of the audio stream in seconds.
 			</description>
 		</method>
+		<method name="instance_playback">
+			<return type="AudioStreamPlayback" />
+			<description>
+    				Returns an AudioStreamPlayback. Useful for when you want to extend `_instance_playback` but call `instance_playback` from another AudioStream.
```

Comment:
```suggestion
    				Returns an [AudioStreamPlayback]. Useful when you want to extend [method _instance_playback] but call [code]instance_playback[/code] from another [AudioStream].
```

The classref uses [special BBCode](https://docs.godotengine.org/en/latest/community/contributing/class_reference_writing_guidelines.html#doc-class-reference-writing-guidelines-bbcode).